common/src/main/java/org/opensearch/ml/common/transport/model_group/MLRegisterModelGroupInput.java

```diff
@@ -17,39 +17,46 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
 import static org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
 
 @Data
 public class MLRegisterModelGroupInput implements ToXContentObject, Writeable{
```

Comment:
If the responsibility of this class to take input for the MLmodel group registration, then may be we can rename it to:
`MLModelGroupRegisterInput`?

Reply:
Just named it so to maintain consistency with model level APIs

Reply:
ok, ignore it then.